README.md

```diff
@@ -1,2 +1,16 @@
 # PvpBot
 killl your discord friends
```

Comment:
Whilst you're editing this file, could you fix the spelling on this?

Reply:
Lol, unless he meant it to be that way, I'll submit another.